business_objects/monitor.py

```diff
@@ -172,7 +180,15 @@ def __select_running_tokenization_tasks(
     limit_per_task: int = 100,
 ) -> str:
     query = f"""
-    SELECT id, '{enums.TaskType.TOKENIZATION.value}' task_type, state, project_id, user_id created_by
+    SELECT id, '{enums.TaskType.TOKENIZATION.value}' task_type, state, project_id, user_id created_by, started_at, finished_at,
+    CASE
+        WHEN type = '{enums.TokenizerTask.TYPE_DOC_BIN.value}' THEN 'Tokenization - docbins - '
+        ELSE 'Tokenization - rats - '
+    END ||
+    CASE
+        WHEN scope = '{enums.TokenizationTaskTypes.PROJECT.value}' THEN 'Project'
+        ELSE attribute_name
+    END AS full_name
```

Comment:
i was expecting this part as task_type clumn since extending NULL for all others is imo complicated